amenity-points.mss

```diff
@@ -553,6 +553,18 @@
     }
   }
 
+  [feature = 'man_made_telescope']["telescope:type" = 'optical'],["telescope:type" = null] {
```

Comment:
This should probably look like this:

  [feature = 'man_made_telescope'][telescope:type = 'optical'],
  [feature = 'man_made_telescope'][telescope:type = null] {

because AFAIK this would check if some object has "telescope:type" = null without checking if this is telescope at all, which would lead to strange problems. 

Also I don't think we need adding quotation marks for keys.

Reply:
The only key we need quotations is "natural", but this is just because this is a PostgreSQL keyword too by accident.